dist/js/tether.js

```diff
@@ -1177,11 +1177,14 @@ var TetherClass = (function (_Evented) {
         }
       };
 
+      var hasOptimizations = typeof this.options.optimizations !== 'undefined';
+      var positionFixed = hasOptimizations ? this.options.optimizations.positionFixed : true;
+
       var moved = false;
       if ((same.page.top || same.page.bottom) && (same.page.left || same.page.right)) {
         css.position = 'absolute';
         transcribe(same.page, pos.page);
-      } else if ((same.viewport.top || same.viewport.bottom) && (same.viewport.left || same.viewport.right)) {
+      } else if (positionFixed && (same.viewport.top || same.viewport.bottom) && (same.viewport.left || same.viewport.right)) {
```

Comment:
I think the `positionFixed` var should be called `allowPositionFixed`, as it doesn't force position fixed, just makes it available.